pkgs/by-name/gv/gvisor/package.nix

```diff
@@ -23,6 +24,19 @@ buildGoModule {
     hash = "sha256-idgUEbYAfnm/HphVs12Sj1FwG+jmL2BBr0PJnG9BC3A=";
   };
 
+  patches = [
+    # gVisor tries to use /sbin/ldconfig, which is not available in NixOS. This
+    # patch inserts a placeholder for the path to ldconfig, which later is replaced
+    # with the actual path to ldconfig from the glibc package.
+    ./0001-container-fix-ldconfig-path.patch
+  ];
+
```

Comment:
```suggestion
```

I think this can be simplified, since the gvisor here already needs to run with some nix dependencies (so no fallback to `/sbin/ldconig` or `/sbin/ldconig.real` is required.

Also see the other comment.